src/main/groovy/org/pih/warehouse/core/Constants.groovy

```diff
@@ -91,7 +91,6 @@ class Constants {
     static final String PRODUCT_INVENTORY_TRANSACTION_TYPE_ID = "11"
     static final String CYCLE_COUNT_PRODUCT_INVENTORY_TRANSACTION_TYPE_ID = "12"
     static final String CYCLE_COUNT_ADJUSTMENT_TRANSACTION_TYPE_ID = "13"
-    static final String CYCLE_COUNT_ADJUSTMENT_DEBIT_TRANSACTION_TYPE_ID = "14"
```

Comment:
https://openboxes.slack.com/archives/C06JCHWR807/p1739986744402599

Reply:
Oh yeah we don't necessarily want to delete the DEBIT adjustment, just standardize on the use case where we use the CREDIT adjustments and allow the sign of the quantity data to dictate whether it's a credit or debit.

Reply:
Oh sorry, this was the cycle count adjustment debit, not the original one.